src/core/vote_accounts.zig

```diff
@@ -394,6 +396,16 @@ pub fn createVoteAccount(
     );
     defer vote_state.deinit();
 
+    try vote_state.votes.append(
+        LandedVote{
```

Comment:
Currently re-reviewing this, if this is the best spot or if it can be moved to the test fixture.